### ISSUE TITLE
howto: compile on Windows

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#98]
"if it is feasible you could maybe write a short manual on how to compile for windows. Or ask Rob if could publish some of the scripts?" 

"Don't know maybe there is a reason for not self compiling on windows but it would be great for testing now and then."

cc @denravonska @gridcoin

**Comment by denravonska** [comment]
We have a wiki somewhere, right? It should cover building for all the platforms in the future.

**Comment by Erkan-Yilmaz** [comment]
there's also:
[WINDOWS BUILD NOTES](https://github.com/gridcoin/Gridcoin-Research/blob/master/doc/build-msw.txt) in the sources

can someone verify that building works with that ?

**Comment by denravonska** [comment]
@Erkan-Yilmaz I remember seeing that as well but couldn't get it running in msys2. Though these layers (msys, cygwin, msys2 etc) for Windows confuse me so it might as well be my fault.

**Comment by gridcoin** [comment]
Yeah, building on windows is not as trivial as some bashers say it is.  Some of the windows dependencies have to be built in a certain way to be compatible (IE noobs would have trouble compiling BOOST with the right flags to even get the windows dlls to be valid and in the right target directory etc).  

I am planning on giving Denravonska my entire build zip library that should remove the ambuiguity on building this current version of Gridcoin on windows.  Im going to do this definitely over the next few days as this is the next high priority item for me to at least give him the ability to build the windows MSI.

In the mean time, I wanted to mention, one quick way to resolve about 80% of the windows build problems is search google for Building Litecoin on Windows.  Since litecoin has a large support base that actually gets one very close as it resolves the windows dependencies.

**Comment by LokiToki** [comment]
I tried to build the wallet on windows from scratch. Downloaded newest and/or needed versions of boost, OpenSSL etc, but i didn't make it in 2 days (effective 4 - 5 hours). There are so many different (as mentioned) layers and libraries...it drove me crazy af tbh. ^^ If there is a high need i would try it again and write everything down noob-style but it will take a bit (very busy atm).

**Comment by Erkan-Yilmaz** [comment]
see also:
* [How to Compile the VB.NET (Neural Network) Code on Windows 10](https://github.com/gridcoin/Gridcoin-Research/wiki/How-to-Compile-the-VB.NET-%28Neural-Network%29-Code-on-Windows-10)
